nixos/tests/paperless.nix

```diff
@@ -25,6 +42,11 @@ import ./make-test-python.nix ({ lib, ... }: {
     def test_paperless(node):
       node.wait_for_unit("paperless-consumer.service")
 
+      # Double check that our attrset option override works as expected
+      cmdline = node.succeed("grep 'paperless-manage' $(systemctl cat paperless-exporter | grep ExecStart | cut -f 2 -d=)")
+      print(f"Exporter command line {cmdline!r}")
+      assert cmdline.strip() == "./paperless-manage document_exporter /var/lib/paperless/exports --compare-checksums --delete --no-progress-bar --no-thumbnail", "Unexpected exporter command line"
```

Comment:
👌

Reply:
> "If it ain't tested, it's broken." (Tres Seaver)

The amount of time it takes us to figure out the right way to do this warrants testing it to avoid an incomprehensible regression ... ;)